### PR TITLE
Only display Standings for years that have races with points.

### LINKED ISSUE #54: Standings page is broken for 2011 results
Just added the first (non-points) race for 2011.

Somehow it broke the standings page:

image

### PR DESCRIPTION
**Pull request by jyundt**
Fixes #54

### CODE REVIEW

File: app/main/views.py [hunk 1]
```diff
@@ -163,7 +163,9 @@ def index():
 @main.route('/standings/')
 def standings():
     years = sorted(set(
-        int(date.year) for (date,) in Race.query.with_entities(Race.date).all()),
+        int(date.year) for (date,) in Race.query.with_entities(Race.date)\
```
Comment (johnwheffner) [inline app/main/views.py]: Style nit: the backslashes are superfluous.  PEP-8 checkers complain about this.

File: app/main/views.py [hunk 2]
```diff
@@ -163,7 +163,9 @@ def index():
...
+                                                .filter_by(points_race=True)\
+                                                .all()),
```
Comment (johnwheffner) [inline app/main/views.py]: Unrelated to this change, but the all() isn't necessary either -- creating the set takes care of evaluating the query.

**Review by johnwheffner — APPROVED** [review]

File: app/main/views.py (hunk 1 above)
Comment (jyundt) [reply app/main/views.py]: Good catch, I'll clean this up.

File: app/main/views.py (hunk 2 above)
Comment (jyundt) [reply app/main/views.py]: Ah, I didn't know that.

**Pull request merged**